mpf/platforms/interfaces/segment_display_platform_interface.py

```diff
@@ -79,3 +79,16 @@ def set_text(self, text: str, flashing: FlashingType) -> None:
     def _set_text(self, text: str) -> None:
         """Set a text to the display."""
         raise NotImplementedError
+
+
+class ColorSegmentDisplayPlatformInterface(SegmentDisplayPlatformInterface):
```

Comment:
Not sure if we want this is segment_display_player calls this unconditionally. What do you think? Either we add this to all segment displays (which probably would not hurt) or we need to check in the player